### PR TITLE
gh-115999: Move specializer test from test_dis to test_opcache

### PR DESCRIPTION
**Pull request by corona10**
* Issue: gh-115999

### CODE REVIEW

**Comment by corona10** [comment]
Motivation: https://github.com/python/cpython/pull/126450#discussion_r1830705866

File: Lib/test/test_opcache.py [hunk 1]
```diff
@@ -1200,5 +1207,55 @@ def f(o, n):
...
+    @requires_specialization_ft
+    def test_contain_op(self):
+        def f():
+            for _ in range(100):
+                a, b = 1, {1: 2, 2: 5}
+                self.assertTrue(a in b)
+                self.assertFalse(3 in b)
+
+        f()
+        self.assert_specialized(f, "CONTAINS_OP_DICT")
```
Comment (corona10) [inline Lib/test/test_opcache.py:1245]: In the future, if the opcodes become more complex, we may need to count how many specific opcodes have occurred.

**Review by markshannon — APPROVED** [review]
Thanks.

We have a lot of tests in `test_dis` that need to be moved elsewhere. Every test moved helps.

**Review by mpage — APPROVED** [review]
LGTM!

**Pull request merged**